README.md

```diff
@@ -100,30 +126,38 @@ threescale_portal_endpoint: <THREESCALE_PORTAL_ENDPOINT>
 threescale_services: ""
 ```
 
+Injector host’s hardware resources should not be performance tests bottleneck. Enough cpu, memory and network resources should be available.
+There are two injector services. The controller has one instance. The agent(s) may have 1 or many. 
+
+*Installation Step*:
+
 **3.** Execute the playbook `injector.yml` to deploy injector.
 
 ```bash
+cd deployment/
 ansible-playbook -i hosts injector.yml
 ```
 
+Managed node host:
+* Docker >= 1.12
+* python >= 2.6
+* docker-py >= 1.7.0
+* ansible >= 2.3.1.0
+
+
 ### Setup traffic profiles
 
-**Steps**:
+*Configuration Steps*:
 
 **1.** Edit the *ansible_host* parameter by replacing **<injector_host>** with the host IP address/DNS name of the host where you want to install the injector component. For example:
```

Comment:
Check step 1 of the `Test your 3scale services` section. Should be the same.

Reply:
It sounds like you don't like the steps being categorized into configuration or installation steps. The changes will be backed out.